source/code-snippets/usage-examples/find.ts

```diff
@@ -0,0 +1,39 @@
+import { MongoClient } from "mongodb";
+
+// Replace the uri string with your MongoDB deployment's connection string.
+const uri =
+  "mongodb+srv://<user>:<password>@<cluster-url>?writeConcern=majority";
+
+const client = new MongoClient(uri);
+
+interface Movies {
+  title: string;
+  imdb: object;
```

Comment:
This should have been a linter warning - avoid "object". Did you mean `Record<string, unknown>`? (Or better yet, something more specific... imdb url?)

>Don't use `object` as a type. The `object` type is currently hard to use ([see this issue](https://github.com/microsoft/TypeScript/issues/21732)).
Consider using `Record<string, unknown>` instead, as it allows you to more easily inspect and use the keys.

Reply:
Woah cool issue, definitely don't understand everything about why using the object type is discouraged, but the length and depth of that discussion says something about it being hard to use. imdb fields are small so I'll just make an interface for them.